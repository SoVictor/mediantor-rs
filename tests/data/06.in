121
1 -655028015
1 943908204
0
1 584556596
1 -895177291
0
0
1 -88225141
0
1 -317702105
1 -45240243
1 945087733
0
0
1 -319929919
0
1 -848749161
1 -449386413
0
1 -844746718
0
0
1 -946489062
1 671366890
0
0
0
0
1 132384146
1 -512424500
1 -96924818
0
1 662704645
1 928282595
0
1 578770804
0
1 453158311
1 423354100
1 206995772
1 -446551651
1 -665327874
0
1 -484016072
1 127047205
1 -419178530
1 -981429781
0
0
0
1 512560968
0
1 -496578805
0
1 658540378
1 464411449
1 27200893
1 976333812
1 680813925
0
0
0
1 -925002585
1 448918791
0
0
1 -210891500
1 417802541
0
0
1 -619382379
0
0
1 95878650
1 -210052682
0
0
0
0
0
1 904684615
0
1 653110988
1 -129948264
1 -456176571
0
0
0
0
1 6147215
0
0
1 -30387453
0
0
0
1 -23253271
1 -985624813
0
1 25907864
0
1 56184454
1 -613599496
0
1 253418906
0
0
0
0
0
1 586995966
0
1 767675580
1 -892475452
0
0
1 -182504701
0
1 -742229586
1 -343351492
1 389399456
