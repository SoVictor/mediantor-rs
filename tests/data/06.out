-655028015
584556596
-895177291
-88225141
-45240243
943908204
-317702105
-449386413
-844746718
-319929919
-848749161
671366890
-946489062
945087733
-96924818
132384146
578770804
206995772
-419178530
-446551651
127047205
423354100
-484016072
464411449
453158311
512560968
27200893
448918791
-210891500
417802541
-496578805
-512424500
95878650
-210052682
658540378
-619382379
662704645
680813925
-129948264
-456176571
653110988
-665327874
6147215
904684615
-30387453
-925002585
928282595
-981429781
-23253271
25907864
56184454
-613599496
253418906
-985624813
976333812
586995966
-892475452
767675580
-182504701
